src/main/java/com/openlattice/search/PersistentSearchApi.java

```diff
@@ -0,0 +1,41 @@
+package com.openlattice.search;
+
+import com.openlattice.search.requests.PersistentSearch;
+import com.openlattice.search.requests.SearchConstraints;
+import retrofit2.http.*;
+
+import java.time.OffsetDateTime;
+import java.util.UUID;
+
+public interface PersistentSearchApi {
+
+    String SERVICE    = "/datastore";
+    String CONTROLLER = "/persistentsearch";
+    String BASE       = SERVICE + CONTROLLER;
+
+    String ALL = "/all";
+
+    String ID         = "id";
+    String EXPIRATION = "/expiration";
+
+    String ID_PATH         = "/{" + ID + "}";
+    String EXPIRATION_PATH = "/{" + EXPIRATION + "}";
+
+    @POST( BASE )
+    UUID createPersistentSearch( @Body PersistentSearch search );
+
+    @GET( BASE )
+    Iterable<PersistentSearch> loadCurrentPersistentSearches();
```

Comment:
Combine this and the other call so you can delete the all path. Make it query parameter to return current or all.